src/utils/ExtractTestResults.ts

```diff
@@ -33,7 +33,7 @@ export const extractMCTestResults = (record: DynamoDBRecord): MCRequest[] => {
   );
   const mcRequest: MCRequest[] = testResultUnmarshall.testTypes
     .filter((testType) =>
-      PROHIB_CLEARANCE_TEST_TYPE_IDS.IDS.includes(testType.testTypeId),
+      PROHIBITION_CLEARANCE_TEST.IDS.includes(testType.testTypeId),
```

Comment:
For consistency across the services, can you use the helper method please. That way, if the logic ever changes, it only needs to change in commons vs every repo that does the `.IDS.includes`